dashboard/src/App.js

```diff
@@ -101,6 +101,10 @@ const App = () => {
                       element={<ComparisonComponent />}
                     />
                   </Route>
+                  <Route
+                    path={APP_ROUTES.ALL_RUNS}
+                    element={<ComingSoonPage />}
+                  />
```

Comment:
A philosophical question rather than a specific complaint on the code: but I've never really made up my mind what "all runs" means vs "results". Do you have a particular concept in mind? I could make an argument, I think, for these being synonyms. While we might imagine slightly distinct semantics regarding the filter pattern (e.g., `access: "public"` vs `mine` vs no filter), I could also see that being represented as a toggle switch on a single view, since we otherwise want the same behavior anyway: and maybe even just subsumed into a general filter mechanism that includes metadata filters.

In the short term, what's the benefit of keeping this unimplemented and potentially confusing sidebar link vs just removing it?

Reply:
Same here:  do we really want to have two separate pages for these, or would we rather have a single view which shows results and which has nice filtering capabilities so that we can choose what we see (e.g., _all_ runs vs. _my_ runs vs. runs with specific characteristics)?

Reply:
Got it. I'll work on it in the upcoming PR